modules/raft/src/main/java/org/apache/ignite/raft/jraft/entity/RaftOutter.java

```diff
@@ -57,6 +57,10 @@ public interface SnapshotMeta extends Message {
 
         long lastIncludedTerm();
 
+        long cfgIndex();
```

Comment:
Do we need these fields for all raft groups or just for partitions (or for partitions + metastorage)? If not for all of them, then probably they don't need to be added here as they are not part of the Raft protocol. If only partitions need them, `PartitionSnapshotMeta` seems to be a more suitable place to add these fields.

Reply:
discussed in private